sdk/translation/ai-translation-document-rest/src/documentTranslationClient.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getClient, ClientOptions } from "@azure-rest/core-client";
+import { logger } from "./logger";
+import { TokenCredential, KeyCredential } from "@azure/core-auth";
+import { DocumentTranslationClient } from "./clientDefinitions";
+
+/**
+ * Initialize a new instance of `DocumentTranslationClient`
+ * @param endpointParam - Supported document Translation endpoint, protocol and hostname, for example: https://{TranslatorResourceName}.cognitiveservices.azure.com/translator.
+ * @param credentials - uniquely identify client credential
+ * @param options - the parameter for all optional parameters
+ */
+export default function createClient(
+  endpointParam: string,
+  credentials: TokenCredential | KeyCredential,
+  options: ClientOptions = {},
+): DocumentTranslationClient {
+  const endpointUrl = options.endpoint ?? options.baseUrl ?? `${endpointParam}/translator`;
+  options.apiVersion = options.apiVersion ?? "2024-05-01";
```

Comment:
we are releasing preview SDK with stable service version?

Reply:
yes